addons/AdvancedRappelling/stringtable.xml

```diff
@@ -3,15 +3,24 @@
     <Package name="Advandced_Rappeling">
         <Key ID="STR_TF47_Rappel_Rappel_Self">
             <English>Rappel Self</English>
-            <German>Abseilen</German>
```

Comment:
hier hast du die deutsche translation gekillt?

Reply:
öhm nö ist noch drin

Reply:
in der Tat, Mystisch das die Zeile weg ist. Laut dem File https://github.com/TaskForce47/TF47-client-mod/blob/update/addons/AdvancedRappelling/stringtable.xml ist irgendwie nur die Formatierung ein wenig durcheinander.